ocrd_models/ocrd_page_user_methods/get_UnorderedGroupChildren.py

```diff
@@ -0,0 +1,7 @@
+def get_UnorderedGroupChildren(self):
+    """
+    List all non-metadata children of an UnorderedGroup
+    """
+    # TODO: should not change order
```

Comment:
Note: IMO it's just not possible to keep the order after parsing. There's no descriptive datum that could preserve the XML ordering of the unordered group. – If that's what you meant by the Todo.

Reply:
I guess you're right, it is an **un**ordered group, after all. But still, for reproducability and non-invasiveness, I would prefer preserving the document order. It's not a blocking issue tho.